metrotranscom/doorway
# feat: expire app data

# Pull Request Template

## Issue Overview

This PR addresses #492


## Description

This PR does the frontend changes for the ticket above which should remove access to application data for partners and jurisdictional admins 45 days after the listing closes. Does not include automated email notifications.

## How Can This Be Tested/Reviewed?

Login as a partner or jurisdictional admin and view the applications table page of a closed listing. You should see the message that notifies you of when the application data expires, which should be 45 days after the closing date. You can manually update closedAt to be more than 45 days in the past to see the empty state. An admin role experience should have no changes.

This changeset is an excellent candidate for a new unit test file, but I dug into trying to configure it and I think it might take a full day, and want to set aside that time later in the sprint to be sure we have capacity.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
